lib/charms/layer/nodejs.py

```diff
@@ -1,8 +1,12 @@
 import os
 import sys
+
 from shell import shell
+from charms import layer
```

Comment:
It is, but layer also has the options for parsing stuff from layer.yaml, but I refactored that code out and need to remove this import now